README.md

```diff
@@ -21,10 +21,20 @@ Although the initial state of this app is very basic, it's work in progress and
 
 ![Scout Dashboard](dashboard.png)
 
-## Contributing
+## Heroku Deploy
+
+Before you start deploying your App using the Heroku Deploy Button. You need to get the nexmo credentials from [Vonage dashboard](https://dashboard.nexmo.com/). Using the firebase console get the [Firebase database secrets](https://firebase.google.com/) and from google cloud get the [google client id](https://console.cloud.google.com/apis/credentials) needed for auth using the client. Two of this credentials are **Files**, the `firebase database secrets` and the `application private key`. You need to put them temporary in your repo to perform the deploy. After that you can remove them.
```

Comment:
There should be a way to not expose private keys or credentials.

Reply:
Perhaps the contents of those files to be pasted in those fields instead if those variables are left blank?